apps/tangle-dapp/data/restake/useRestakeDelegatorInfo.ts

```diff
@@ -56,9 +32,7 @@ export default function useRestakeDelegatorInfo() {
           }
 
           return apiRx.query.multiAssetDelegation
-            ?.delegators<
-              Option<PalletMultiAssetDelegationDelegatorDelegatorMetadata>
-            >(activeAddress ?? '')
+            ?.delegators(activeAddress ?? '')
```

Comment:
What would happen when `delegators` is called with an empty string, when the active address isn't present? Might be better to explicitly handle this edge case, in case that empty string poses issues

Reply:
The empty string returns `Option.None`, but I agree that I need to handle the case when `activeAddress === null`.